packages/cookie-banner/src/lib/utils.js

```diff
@@ -113,7 +113,7 @@ export const uuidv4 = () => 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]
     return v.toString(16);
 });
 
-export const getFocusableChildren = node => [].slice.call(node.querySelectorAll(FOCUSABLE_ELEMENTS.join(',')));
+export const getFocusableChildren = node => [].slice.call(node.querySelectorAll(FOCUSABLE_ELEMENTS.join(','))).filter(el => el.offsetWidth > 0 || el.offsetHeight > 0);
```

Comment:
I've made a small update to get rid of the usesToggle and any references to the toggle, instead the getFocusableChildren gets and filters all children of the banner based on visibility so anything behind a toggle is only listed when the toggle is open.